### DISCUSSION TITLE
live trial - component on the page

### DISCUSSION THREAD

**Discussion by benkoshy** [benkoshy/pagy#16] [Ideas]
@ddnexus 

### What is pagy?

Is it a ridiculous idea to have a "Live" version of Pagy right there on the landing page - so people can paginate within their retype docs to see how this works? i.e. some type of iframe, or some javascript component to demo pagy without downloading anything?

i.e. a pagy equivalent of the following: https://getbootstrap.com/docs/5.2/components/toasts/#live-example

![image](https://user-images.githubusercontent.com/15097447/178857942-2bef451f-eff8-4db6-a13b-3e02557a29d7.png)

Probably many other things which need doing before such ideas, but anyways my two cents.

**Comment by ddnexus** [comment]
OK, low priority as you said. The release will start without it, but it could be added later.
